config/locales/en.yml

```diff
@@ -1482,6 +1482,8 @@ en:
     new:
       header: We'd love to work with you next year to help you file your taxes or claim your stimulus for free!
       subheader: Please sign up here to receive a notification when we open in January 2022.
+      opt_in_message: "Note: Providing your phone number means you have opted in to the following in order to use the GetYourRefund service: Receiving a GetYourRefund Verification Code Tax return notifications. Message frequency will vary. Standard message rates apply. Text HELP to 11111 for help. Text STOP to 11111 to cancel. Carriers (e.g. AT&T, Verizon, etc.) are not responsible or liable for undelivered or delayed messages."
+      detail_links_html: See our <a href=%{terms_link} target="_blank" rel="noopener noreferrer">Terms and Conditions</a> and <a href=%{privacy_link} target="_blank" rel="noopener noreferrer">Privacy Policy.</a>
```

Comment:
It's good to wrap this in quotation marks: `<a href="%{terms_link}">` not `<a href=%{terms_link}>`.

Reply:
Same w/ privacy_link